src/cfn.py

```diff
@@ -313,11 +315,15 @@ def download_file(stackname, path, destination='.', node=None, allow_missing="Fa
     Boolean arguments are expressed as strings as this is the idiomatic way of passing them from the command line.
     """
     allow_missing, use_bootstrap_user = lmap(strtobool, [allow_missing, use_bootstrap_user])
-    with stack_conn(stackname, username=BOOTSTRAP_USER if use_bootstrap_user else DEPLOY_USER, node=node):
-        if allow_missing and not files.exists(path):
-            return # skip download
-        get(path, destination, use_sudo=True)
 
+    @backoff.on_exception(backoff.expo, fabric.exceptions.NetworkError, max_time=60)
```

Comment:
can't hurt I suppose ...

Reply:
While I was testing this fix, this other error on `download_file` in the `integration_test.test_provisioning` module came up. If a newly created instance doesn't let you access it through SSH yet... never give up, never surrender.